documentation/asciidoc/computers/camera/libcamera_apps_getting_help.adoc

```diff
@@ -2,9 +2,7 @@
 
 For further help with `libcamera` and the `libcamera-apps`, the first port of call will usually be the https://forums.raspberrypi.com/viewforum.php?f=43[Raspberry Pi Camera Forum]. Before posting, it's helpful to:
 
-* Ensure your xref:../computers/os.adoc#using-apt[software is up to date].
-
-* If you are using _Buster_ please upgrade to the latest OS, as `libcamera-apps` is no longer supported there.
+* Make sure your xref:../computers/os.adoc#using-apt[operating system is up to date] as some legacy versions of the Raspberry Pi OS do not support `libcamera-apps`.
```

Comment:
This can probably be removed, as both Bullseye (which will be Legacy) and Bookworm fully support libcamera-apps.

Reply:
Yup. The entire `raspicam` section in the camera documentation is getting yanked, see #2960. In fact all reference to the legacy stack should have already been snipped from the `bookworm` branch, as #2938 seems to have been merged already.

Reply:
Done